### ISSUE TITLE
Comma in password

### ISSUE THREAD

**Issue by mitasov-ra** [provectus/kafka-ui#1928]
**Describe the bug**
If i connect with SASL authentification and I have comma in password then authentification is failed.

**Steps to Reproduce**

```
docker run -p 8080:8080 \
    -e KAFKA_CLUSTERS_0_NAME=test \
    -e KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS='test:9093' \
    -e KAFKA_CLUSTERS_0_PROPERTIES_SECURITY_PROTOCOL=SASL_PLAINTEXT \
    -e KAFKA_CLUSTERS_0_PROPERTIES_SASL_MECHANISM=SCRAM-SHA-512 \
    -e KAFKA_CLUSTERS_0_PROPERTIES_SASL_JAAS_CONFIG='org.apache.kafka.common.security.scram.ScramLoginModule required username="test" password="password,with,comma";' \
    -d provectuslabs/kafka-ui:latest
```

**Expected behavior**

Everything works as normal

**Comment by Haarolean** [comment]
Hey, thanks for reaching out. We'll take a look into this.

**Comment by Haarolean** [comment]
@mitasov-ra can you provide an example setup for kafka itself to reproduce the problem?

**Comment by mitasov-ra** [comment]
Hi! Unfortunately, I'm not able to show our cluster's configs due to NDA.

But I can show you the error log:

```
2022-05-26 14:09:10,114 INFO  [kafka-admin-client-thread | adminclient-1] o.a.k.c.a.i.AdminMetadataManager: [AdminClient clientId=adminclient-1] Metadata update failed
org.apache.kafka.common.errors.TimeoutException: Call(callName=fetchMetadata, deadlineMs=1653574177906, tries=1, nextAllowedTryMs=-9223372036854775709) timed out at 9223372036854775807 after 1 attempt(s)
Caused by: org.apache.kafka.common.errors.TimeoutException: Timed out waiting to send the call. Call: fetchMetadata
2022-05-26 14:09:10,121 ERROR [parallel-1] c.p.k.u.s.MetricsService: Failed to collect cluster dev.kafka_cluster info
java.lang.IllegalStateException: Error while creating AdminClient for Cluster dev.kafka_cluster
	at com.provectus.kafka.ui.service.AdminClientServiceImpl.lambda$createAdminClient$3(AdminClientServiceImpl.java:45)
	at reactor.core.publisher.Mono.lambda$onErrorMap$31(Mono.java:3733)
	at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:94)
	at reactor.core.publisher.FluxMapFuseable$MapFuseableSubscriber.onError(FluxMapFuseable.java:140)
	at reactor.core.publisher.MonoFlatMap$FlatMapMain.onError(MonoFlatMap.java:172)
	at reactor.core.publisher.MonoPublishOn$PublishOnSubscriber.run(MonoPublishOn.java:187)
	at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:68)
	at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:28)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
Caused by: org.apache.kafka.common.errors.TimeoutException: Call(callName=listNodes, deadlineMs=1653574207923, tries=1, nextAllowedTryMs=-9223372036854775709) timed out at 9223372036854775807 after 1 attempt(s)
Caused by: org.apache.kafka.common.errors.TimeoutException: The AdminClient thread has exited. Call: listNodes
2022-05-26 14:09:10,142 DEBUG [parallel-1] c.p.k.u.s.ClustersMetricsScheduler: Metrics updated for cluster: dev.kafka_cluster
2022-05-26 14:09:10,159 INFO  [kafka-admin-client-thread | adminclient-1] o.a.k.c.m.Metrics: Metrics scheduler closed
2022-05-26 14:09:10,160 INFO  [kafka-admin-client-thread | adminclient-1] o.a.k.c.m.Metrics: Closing reporter org.apache.kafka.common.metrics.JmxReporter
2022-05-26 14:09:10,160 INFO  [kafka-admin-client-thread | adminclient-1] o.a.k.c.m.Metrics: Metrics reporters closed
2022-05-26 14:09:10,160 ERROR [kafka-admin-client-thread | adminclient-1] o.a.k.c.u.KafkaThread: Uncaught exception in thread 'kafka-admin-client-thread | adminclient-1':
java.lang.OutOfMemoryError: Java heap space
	at java.base/java.nio.HeapByteBuffer.<init>(HeapByteBuffer.java:63)
	at java.base/java.nio.ByteBuffer.allocate(ByteBuffer.java:351)
	at org.apache.kafka.common.memory.MemoryPool$1.tryAllocate(MemoryPool.java:30)
	at org.apache.kafka.common.network.NetworkReceive.readFrom(NetworkReceive.java:113)
	at org.apache.kafka.common.security.authenticator.SaslClientAuthenticator.receiveResponseOrToken(SaslClientAuthenticator.java:475)
	at org.apache.kafka.common.security.authenticator.SaslClientAuthenticator.receiveKafkaResponse(SaslClientAuthenticator.java:572)
	at org.apache.kafka.common.security.authenticator.SaslClientAuthenticator.authenticate(SaslClientAuthenticator.java:250)
	at org.apache.kafka.common.network.KafkaChannel.prepare(KafkaChannel.java:181)
	at org.apache.kafka.common.network.Selector.pollSelectionKeys(Selector.java:543)
	at org.apache.kafka.common.network.Selector.poll(Selector.java:481)
	at org.apache.kafka.clients.NetworkClient.poll(NetworkClient.java:561)
	at org.apache.kafka.clients.admin.KafkaAdminClient$AdminClientRunnable.processRequests(KafkaAdminClient.java:1333)
	at org.apache.kafka.clients.admin.KafkaAdminClient$AdminClientRunnable.run(KafkaAdminClient.java:1264)
	at java.base/java.lang.Thread.run(Thread.java:830)

**Comment by Haarolean** [comment]
@mitasov-ra oh wow, I'm not quite sure it's related to the password. Wanna chat [on discord](https://discord.gg/4DWzD7pGE5) to make it quicker?

**Comment by Haarolean** [comment]
To be fixed within #2116
